examples/otlp-exporter-node/metrics.js

```diff
@@ -36,6 +61,11 @@ const upDownCounter = meter.createUpDownCounter('test_up_down_counter', {
 
 const histogram = meter.createHistogram('test_histogram', {
   description: 'Example of a Histogram',
+  boundaries: [0, 10, 100, 1000],
```

Comment:
I think we don't have that implemented yet :/
```suggestion
```

Reply:
Ahh, yes. It's still being worked at https://github.com/open-telemetry/opentelemetry-js/pull/3876.

Reply:
🤦  good catch, thanks!